inc/class-upgrade.php

```diff
@@ -547,4 +520,40 @@ private function upgrade_70() {
 		global $wpdb;
 		$wpdb->query( "UPDATE $wpdb->usermeta SET meta_key = 'wpseo_noindex_author' WHERE meta_key = 'wpseo_excludeauthorsitemap'" );
 	}
+
+	/**
+	 * Retrieves the option value directly from the database.
+	 *
+	 * @param string $option_name Option to retrieve.
+	 *
+	 * @return array|mixed The content of the option if exists, otherwise an empty array.
+	 */
+	protected function get_option_raw( $option_name ) {
+		global $wpdb;
+
+		// Load option directly from the database, to avoid filtering and sanitization.
+		$sql     = $wpdb->prepare( 'SELECT option_value FROM ' . $wpdb->options . ' WHERE option_name = %s', $option_name );
+		$results = $wpdb->get_results( $sql, ARRAY_A );
+		if ( ! empty( $results ) ) {
+			return maybe_unserialize( $results[0]['option_value'] );
+		}
+
+		return array();
+	}
+
+	/**
+	 * Re-save the option to make sure only the expected values are there.
+	 *
+	 * @param string $option_name Option name to sanitize.
+	 *
+	 * @return void
+	 */
+	protected function refresh_option_content( $option_name ) {
```

Comment:
I see 3 different purposes around this method:
1. In de description it is re-saving the option
1. The method name says it refreshes the options
1. The param description let me think it is sanitizing the option.

When I look at that content of the method it is retrieving the option and saves it with the same value when it is a filled array. 

I have seriously no clue what the purpose of this method might be. What is happening?